main.go

```diff
@@ -1024,17 +1024,24 @@ func getBuildParameters(conf *config, build *buildOptions) ([]*gitlab.PipelineVa
 		}
 	}
 
-	// set the correct integraton branches if we aren't performing a pull request again integration
+	// set the correct integration branches if we aren't performing a pull request again integration
 	if build.repo != "integration" && build.repo != "meta-mender" {
 		buildParameters = append(buildParameters, &gitlab.PipelineVariable{Key: repoToBuildParameter("integration"), Value: build.baseBranch})
 	}
 
-	// set the poky branch equal to the meta-mender base branch, unless it
-	// is master, in which case we rely on the default.
+	// set the poky branch equal to the meta-mender base branch, unless:
+	// * It is master, in which case we rely on the default, or
+	// * It is master-next, in which case we use poky master
 	if build.repo == "meta-mender" && build.baseBranch != "master" {
-		buildParameters = append(buildParameters, &gitlab.PipelineVariable{Key: repoToBuildParameter("poky"), Value: build.baseBranch})
-		buildParameters = append(buildParameters, &gitlab.PipelineVariable{Key: repoToBuildParameter("meta-openembedded"), Value: build.baseBranch})
-		buildParameters = append(buildParameters, &gitlab.PipelineVariable{Key: repoToBuildParameter("meta-raspberrypi"), Value: build.baseBranch})
+		var pokyBranch string
+		if build.baseBranch == "master-next" {
+			pokyBranch = "master"
```

Comment:
`mendersoftware/poky:master` is not being kept up to date, because we don't use it. So this will likely require manual action regardless. Is a better approach to simply not start a build?

Reply:
Yes, [I figured that out](https://github.com/mendersoftware/meta-mender/pull/1083#issuecomment-691050866).

Okey, let's not add more maintenance burden at the moment... agreed with just skip the build in these. I will rework this PR.